docs/migration/test-cloud/index.md

```diff
@@ -1,13 +1,94 @@
 ---
-title: "Test Cloud Migration"
-author: joemellin
-ms.author: joemel
-ms.date: 04/10/2017
+title: Test Cloud Migration Guide
+description: "Xamarin Test Cloud to AppCenter test tranition guide. Relevant to Xamarin Test Cloud subscribers that will be automatically upgraded from Xamarin Test Cloud to Visual Studio App Center on March 31, 2018"
+keywords: testcloud
+author: glwest
+ms.author: glwest
+ms.date: 2/21/2018
 ms.topic: article
-ms.assetid: 8495359C-F8E6-45B4-8528-C1EFC0A6B247
+ms.assetid: 4a110ae0-21d6-46c4-bcca-963ac0e2c3d7
 ms.service: vs-appcenter
+ms.custom: test
 ---
 
-# Test Cloud Migration
+# Upgrading all Xamarin Test Cloud subscribers to Visual Studio App Center
+
+Microsoft will seamlessly upgrade active paid Xamarin Test Cloud subscriptions to [Visual Studio App Center](http://appcenter.ms). Visual Studio App Center offers the same devices, performance, and features as Xamarin Test Cloud, plus an upgraded API, unified CLI, and new features like saved device sets. It also integrates our automated test lab with our other App Center services, like cloud-based builds, app distribution, crash reporting, user analytics, and push notifications.
+
+## Timeline
+* March 31st, 2018 
+	1. Teams / Orgs / Apps / previous test reports will import into App Center. 
+	2. All your test results that are being uploaded via our XTC CLIs will automatically route your test results into your App Center account.
```

Comment:
remove "your" and "that are being" (eliminates passive voice) - becomes "all test results uploaded via..." - makes it much simpler.